litrpc/lit-sessions.proto

```diff
@@ -76,6 +76,8 @@ message Session {
     bytes remote_public_key = 10;
 
     uint64 created_at = 11 [jstype = JS_STRING];
+
+    repeated MacaroonPermission macaroon_custom_permissions = 12;
```

Comment:
Should we show the full recipe here? So including the caveats?

Reply:
hmmm do we also want to add `CustomCaveats` to the `AddSessionRequest` then?

Reply:
if we were to add the custom-caveats option to the request, then would it be fine just to let the user specify strings that we will use for the caveat `ID`? ie, is it fine to leave the location & verification ID empty ?